packages/fuel-indexer-tests/src/lib.rs

```diff
@@ -47,17 +47,21 @@ pub mod utils {
     pub fn update_test_manifest_asset_paths(manifest: &mut Manifest) {
         let manifest_dir = Path::new(WORKSPACE_ROOT);
         manifest.graphql_schema = manifest_dir
-            .join("..")
-            .join("..")
+            .parent()
```

Comment:
- Is there a reason for this change? 
- I believe they accomplish the same thing but this requires an additional `.unwrap()`?

Reply:
So the `.join()`s weren't actually referring to the parent in practice; it was just building a string that wasn't resolved correctly. See the error message here -- https://github.com/FuelLabs/fuel-indexer/actions/runs/4146742221/jobs/7172823151:

```
---- e2e::indexing_postgres::test_index_metadata_is_saved_when_indexer_macro_is_called_postgres stdout ----
thread 'e2e::indexing_postgres::test_index_metadata_is_saved_when_indexer_macro_is_called_postgres' panicked at 'Failed to initialize indexer.: IoError(Custom { kind: NotFound, error: VerboseError { source: Os { code: 2, kind: NotFound, message: "No such file or directory" }, message: "could not open `/home/runner/work/fuel-indexer/fuel-indexer/packages/fuel-indexer-tests/../../packages/fuel-indexer-tests/assets/fuel_indexer_test.wasm`" } })', packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs:514:10
```